tutorial/sample-problems/IntegralCalc/GraphShading.pg

```diff
@@ -32,11 +32,13 @@ loadMacros('PGstandard.pl', 'PGML.pl', 'PGtikz.pl', 'PGcourse.pl');
 #: The shading in done with the `\filldraw` command with the
 #: `fill opacity = 0.5`. Also note that one of the sides of the filled area is
 #: the square root function.
-$a    = random(-4, 2);
-$b    = random(1,  4);
-$g    = Compute("$b + sqrt(x - $a)");
-$gtex = "$b + \sqrt{" . nicestring([ 1, -$a ]) . '}';
-$ans  = Compute("3*$b + 14/3");
+
+Context('Numeric');
```

Comment:
Delete this line.  I just finished removing all of these unnecessary calls from the sample problems.  The `Numeric` context is the default context.  There is not need to activate it if that is the context to be used.